bench/bench.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const path = require('path');
+const load = require('load-json-file');
+const Benchmark = require('benchmark');
+const jstsUnion = require('@turf/union');
+const w8r = require('martinez-polygon-clipping');
+const mfogel = require('../src/index');
```

Comment:
I think it would be better if the benchmark went through the api mentioned in the README - which is actually right now in `main.js`. So this would change to

```js
const mfogel = require('../main');
```

Where `mfogel` is then used below in the benchmark suites, it would have the exact same form as `w8r` - ie `mfogel.union(...)` and the `operation` require on the next line would not be needed.